lib/src/main/kotlin/starknet/data/hex.kt

```diff
@@ -12,3 +12,4 @@ fun parseHex(value: String): BigInteger {
 }
 
 fun toHex(value: BigInteger): String = "0x" + value.toString(16)
+fun toHex(value: Int): String = "0x" + value.toString(16)
```

Comment:
These could probably work as extensions methods, but that may be an issue for possible future refactors.

Reply:
Duplicate